pkg/fetcher/fetcher.go

```diff
@@ -100,6 +100,21 @@ type URLFetcher struct {
 	options Options
 }
 
+// ErrorResponseBody struct
+type ErrorResponseBody struct {
+	Errors []struct {
+		Code string
+		Message string
+
+		Detail []struct {
```

Comment:
Is `Detail` always a struct like this? Is it possible that it is just a string? From the documentation, I don't see such definition.